test/hw8/test_task2.py

```diff
@@ -0,0 +1,106 @@
+import os
+import sqlite3
+import tempfile
+import time
+
+import pytest
+
+from All_home_works.hw8.task2 import TableData
+
+
+@pytest.fixture(scope="module")
+def create_db_table():
+    directory = tempfile.TemporaryDirectory()
+    path = os.path.join(directory.name, "test.sqlite")
+    db_name = os.path.normpath(path)
+    db_table = f"test_table_{time.time_ns()}"
+
+    conn = sqlite3.connect(db_name)
```

Comment:
Actually, you could change db_name value and this will simplify your code a lot.